app/Http/Controllers/UserController.php

```diff
@@ -2,12 +2,18 @@
 
 namespace App\Http\Controllers;
 
+use App\User;
 use Illuminate\Http\Request;
 
 class UserController extends Controller
 {
     public function dashboardView()
     {
-        return view('/user/dashboard');
+        $learnedWordsNumber = app(User::class)::find(auth()->id())->learnedWord->count();
+        $learnedLessonsNumber = app(User::class)::find(auth()->id())->learnedLesson->count();
```

Comment:
```
  public function dashboardView()
    {
        $learnedWordsNumber = Auth::user()->learnedWord->count();
        $learnedLessonsNumber = Auth::user()->learnedLesson->count();
        ... some code
```

so you don't need to use `find()` to get the authenticated user, just call that directly an you have an authenticated user including getting it's id.